Apps/WebClient/src/Server/Controllers/DependentController.cs

```diff
@@ -32,7 +32,7 @@ namespace HealthGateway.WebClient.Controllers
     /// </summary>
     [Authorize]
     [ApiVersion("1.0")]
-    [Route("v{version:apiVersion}/api/[controller]")]
+    [Route("v{version:apiVersion}/api/UserProfile/")]
```

Comment:
It would be nice if there was a way of injecting typed controller in here!

Reply:
Yeah! I tried a couple of things but no luck